spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -78,11 +74,6 @@ def assert_common
     assert_network_router
     assert_relationship_tree
     assert_specific_labels_on_vm
-
-    if options.inventory_object_refresh
-      assert_specific_service_offerings
-      assert_specific_service_instances
-    end
```

Comment:
We probably want to test both of these since `options.inventory_object_refresh` should always be true now right?